src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,58 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client errors
+
+HTTP Client errors are automatically captured and sent to `https://sentry.io`.
+
+The HTTP Client integration should have 3 configuration options:
+
+* `captureFailedRequests` defaults to `false` due to PII reasons.
+* `failedRequestStatusCodes` defaults to `500` to `599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
+* `failedRequestTargets` defaults to `.*`, this configuration option accepts a `List` of `String` that may be Regular expressions as well, similar to <Link to="/sdk/performance/#tracepropagationtargets">tracePropagationTargets</Link>.
+
+Then the HTTP Client integration should not capture the error:
+
+* If `captureFailedRequests` is disabled.
+* If the HTTP Request url isn't a match for any of the `failedRequestsTargets`.
+* If the HTTP Response status code isn't within the defined ranges in `failedRequestStatusCodes`.
+
+The HTTP Client integration should capture error events with the following properties:
+
+The Request interface, See the <Link to="/sdk/event-payloads/request/">Spec</Link> for details.
```

Comment:
`l`: I think there is no reason to use capitals for these. The same applies to the next couple of lines.
```suggestion
The request interface, see the <Link to="/sdk/event-payloads/request/">spec</Link> for details.
```

Reply:
Changing `See` to `see`, but will keep `Request`, docs team always keep them as Uppercase.